suite-native/navigation/src/components/Screen.tsx

```diff
@@ -29,24 +31,47 @@ const screenContainerStyle = prepareNativeStyle<{
     insets: EdgeInsets;
     customVerticalPadding: number;
     isTabBarVisible: boolean;
-}>((utils, { backgroundColor, customVerticalPadding, insets, isTabBarVisible }) => ({
-    flex: 1,
-    backgroundColor: utils.colors[backgroundColor],
-    paddingTop: Math.max(insets.top, customVerticalPadding),
-    extend: {
-        condition: !isTabBarVisible,
-        style: {
-            paddingBottom: Math.max(insets.bottom, customVerticalPadding),
+    isMessageBannerDisplayed: boolean;
+}>(
+    (
+        utils,
+        {
+            backgroundColor,
+            customVerticalPadding,
+            insets,
+            isTabBarVisible,
+            isMessageBannerDisplayed,
         },
-    },
-}));
+    ) => ({
+        flex: 1,
+        backgroundColor: utils.colors[backgroundColor],
+        paddingTop: Math.max(insets.top, customVerticalPadding),
+        extend: [
+            {
+                condition: !isTabBarVisible,
+                style: {
+                    paddingBottom: Math.max(insets.bottom, customVerticalPadding),
+                },
+            },
+            {
+                // If the message banner is displayed, the top padding has to be equal to 0
+                // to render the app content right under the banner.
+                condition: isMessageBannerDisplayed,
+                style: {
+                    paddingTop: 0,
```

Comment:
I think you should never use just `0` at least it should be `insets.top` to avoid some hiding stuff under top bar or something.

Reply:
Since this is a conditional style, it will never happen that the content of a screen will be hidden behind the top bar because `paddingTop` will be set to `0` only if there is `MessageBanner` rendered above (It is a sibling component of the `Screen`).

If I set the padding to `insets.top` then there is extra padding (equal to the height of the top bar) between the banner message and the screen component. For this reason, I believe that `0` is a valid value in this case.